lib/radiator_web/controllers/player_controller.ex

```diff
@@ -0,0 +1,80 @@
+defmodule RadiatorWeb.PlayerController do
```

Comment:
just as a work item: we should sit together and unify the keys/values for the config and our graphql schema to make this as frictionless and meaningful as possible.